server/app.js

```diff
@@ -43,6 +44,14 @@ app.use('/auth', authRouter);
 app.use(listRouter);
 app.use(productRouter);
 
+const scrapePriceTag = () => { 
+	console.log("running a task every two minutes");
```

Comment:
two minutes! Will prices change so frequently?

Reply:
Haha yeah that is just added for now because I didn't want to wait for 10 minutes or so. Sure, will change it in final code build up. I read a blog about Amazon which said their prices change every 10 minutes. Any thoughts team?

Reply:
Got it! It's easier for developing and testing. Great!